### ISSUE TITLE
Upgrade Sentry to latest version

### ISSUE THREAD

**Issue by ukch** [fastmonkeys/sentry-on-heroku#13]
According to [Sentry's Github](https://github.com/getsentry/sentry/), the latest version is 7.4.3, but we depend on version 6.4.4. Would upgrading be as simple as bumping the version in requirements.txt or might it cause issues?

**Comment by paulomcnally** [comment]
I try latest and heroku don't support this version. Any idea?

**Comment by ukch** [comment]
I have found this in the README:

> The version of Sentry pinned in the `requirements.txt` file is one of the last for which Redis was not a hard dependency. Newer versions of Sentry will not work on Heroku using this configuration as Redis is required.

**Comment by tucksaun** [comment]
I personally  was able to update to sentry 7.1.1 by adding the `rediscloud` addon (which can be done in `app.json`) and by tweaking `sentry.conf.py` with the following patch:

```diff
+SENTRY_TSDB = 'sentry.tsdb.redis.RedisTSDB'
+redis_url = urlparse.urlparse(os.environ.get('REDISCLOUD_URL'))
+SENTRY_TSDB_OPTIONS = {
+    'hosts': {
+        0: {
+            'host': redis_url.hostname,
+            'port': redis_url.port,
+            'password': redis_url.password
+        }
+    }
+}
```

**Comment by joshkel** [comment]
I was able to update with the help of @tucksaun's `rediscloud` addon and patch. I had to make a couple of additional changes too; see https://github.com/joshkel/sentry-on-heroku.

**Comment by jpvanhal** [comment]
Fixed in commit 52d38498e132f9aa72f9b80f6049023c5c62ed51.
